deployability/modules/testing/models.py

```diff
@@ -14,29 +14,27 @@ class ExtraVars(BaseModel):
 class InputPayload(ExtraVars):
     """Input payload for testing module."""
     tests: list[str]
-    inventory: Path
-    dependencies: list[str] = []
+    targets: list[str]
+    dependencies: list[str]
     cleanup: bool = True
 
+
     @field_validator('tests', mode='before')
     def validate_tests(cls, value) -> list[str]:
         """Validate tests names."""
         if type(value) is str:
             value = value.split(',')
+
         return value
 
-    @field_validator('inventory', mode='before')
-    def validate_inventory(cls, value) -> Path:
-        """Validate inventory path."""
-        if not Path(value).exists():
-            raise ValueError(f'Inventory file "{value}" does not exist')
-        return Path(value)
+    @field_validator('targets', mode='before')
+    def validate_targets(cls, values) -> list:
+        """Validate required fields."""
+
+        return values
```

Comment:
This validator is not adding any value to the field's validation, it must be deleted.